pom.xml

```diff
@@ -129,6 +129,7 @@
     <maven-deploy-plugin.version>3.1.4</maven-deploy-plugin.version>
     <license-plugin.version>2.6.0</license-plugin.version>
     <org-jvnet-localizer-plugin.version>1.31</org-jvnet-localizer-plugin.version>
+    <skipTests>false</skipTests>
```

Comment:
Is this related to the benchmark pom changes? This doesn't seem like it resolves any of the warnings in the related issue.